core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -163,6 +166,19 @@ public class TableTestBase {
 
   static final FileIO FILE_IO = new TestTables.LocalFileIO();
 
+  static final Map<String, String> CODEC_METADATA_MAPPING =
+      ImmutableMap.<String, String>builder()
+          .put("uncompressed", "null")
```

Comment:
can we use the `Avro.Codec` enum here rather than plain strings?

Reply:
I don't see much benefit to using `Avro.Codec`. You'd have to get the name from the enum and then lowercase it. I think in this case, plain strings are fine.